test/run_test.py

```diff
@@ -13,45 +13,61 @@
+sys.stderr.write("Running model refinement (--fit-model refine)\n")
+subprocess.run("python ../poppunk-runner.py --fit-model refine --distances example_db/example_db.dists --ref-db example_db --output example_refine --neg-shift 0.8 --overwrite", shell=True, check=True)
 
-#assign query
-sys.stderr.write("Running query assignment (--assign-query)\n")
-subprocess.run("python ../poppunk-runner.py --assign-query --q-files queries.txt --distances example_db/example_db.dists --ref-db example_db --output example_query --update-db  --qc-filter prune --overwrite", shell=True, check=True)
+# lineage clustering
+sys.stderr.write("Running lineage clustering test (--fit-model lineage)\n")
+subprocess.run("python ../poppunk-runner.py --fit-model lineage --distances example_db/example_db.dists --output example_lineages --ranks 1,2,3,5 --ref-db example_db --overwrite", shell=True, check=True)
 
 #use model
 sys.stderr.write("Running with an existing model (--use-model)\n")
 subprocess.run("python ../poppunk-runner.py --use-model --ref-db example_db --model-dir example_db --distances example_db/example_db.dists --output example_use --overwrite", shell=True, check=True)
 
-#generate viz
-sys.stderr.write("Running microreact visualisations (--generate-viz)\n")
-subprocess.run("python ../poppunk-runner.py --generate-viz --distances example_db/example_db.dists --ref-db example_db --output example_viz --microreact --subset subset.txt", shell=True, check=True)
-
-# general tests
-sys.stderr.write("Running general tests\n\n")
+# tests of other command line programs
 
-# lineage clustering
-sys.stderr.write("Running lineage clustering test (--lineage-clustering)\n")
-subprocess.run("python ../poppunk-runner.py --lineage-clustering --distances example_db/example_db.dists --output example_lineages --ranks 1,2,3,5 --full-db --ref-db example_db --overwrite", shell=True, check=True)
-
-# assign query to lineages
-sys.stderr.write("Running query assignment (--assign-lineages)\n")
-subprocess.run("python ../poppunk-runner.py --assign-lineages --q-files some_queries.txt --distances example_db/example_db.dists --ranks 1,2,3,5 --ref-db example_db --model-dir example_lineages --output example_lineage_query --update-db --qc-filter prune --overwrite", shell=True, check=True)
-
-# tests of other command line programs (TODO)
+#assign query
+sys.stderr.write("Running query assignment\n")
+subprocess.run("python ../poppunk_assign-runner.py --q-files some_queries.txt --distances example_db/example_db.dists --ref-db example_db --output example_query --overwrite", shell=True, check=True)
+subprocess.run("python ../poppunk_assign-runner.py --q-files some_queries.txt --distances example_db/example_db.dists --ref-db example_db --output example_query_update --update-db --overwrite", shell=True, check=True)
```

Comment:
Another superficial comment, but could we convert `--ref-db` to `--db` and `--q-files` to `--query`? Once the command is `poppunk_assign-runner.py --db example_db --query some_queries.txt`, it looks like a much more conventional (and less daunting) bit of software. Maybe too minor to be worthwhile, but from a user intrface point of view, perhaps helpful.

Reply:
Also a good suggestion